lib/fastlane/plugin/wpmreleasetoolkit/actions/common/prototype_build_details_comment_action.rb

```diff
@@ -0,0 +1,130 @@
+module Fastlane
+  module Actions
+    class PrototypeBuildDetailsCommentAction < Action
+      def self.run(params)
+        # Get Input Parameters
+        appcenter_org_name = params[:appcenter_org_name]
+        appcenter_app_name = params[:appcenter_app_name]
+        appcenter_release_id = params[:appcenter_release_id]
+        commit = params[:commit] || other_action.last_git_commit[:abbreviated_commit_hash]
+
+        metadata = params[:metadata] # e.g. {'Build Config':… , 'Version': …, 'Short Version': …}
+        direct_link = params[:download_url]
+        unless direct_link.nil?
+          metadata['Direct Link'] = "<a href='#{direct_link}'><tt>#{File.basename(direct_link)}</tt></a>"
+        end
+
+        # Build the comment parts
+        install_url = "https://install.appcenter.ms/orgs/#{appcenter_org_name}/apps/#{appcenter_app_name}/releases/#{appcenter_release_id}"
+        qr_code_url = "https://chart.googleapis.com/chart?chs=500x500&cht=qr&chl=#{CGI.escape(install_url)}&choe=UTF-8"
+        metadata_rows = metadata.compact.map do |key, value|
+          "<tr><td><b>#{key}</b></td><td>#{value}</td></tr>"
+        end
+
+        intro = "📲 You can test the changes from this Pull Request in <strong>#{appcenter_app_name}</strong> by scanning the QR code below to install the corresponding build via App Center."
+        body = <<~COMMENT_BODY
+          <table>
+          <tr>
+            <td rowspan='#{metadata.count + 3}'><img src='#{qr_code_url}' width='250' height='250' /></td>
+            <td width='150px'><b>App Name</b></td><td><tt>#{appcenter_app_name}</tt></td>
+          </tr>
+          #{metadata_rows.join("\n")}
+          <tr><td><b>App Center Build</b></td><td><a href='#{install_url}'>Build \##{appcenter_release_id}</a></td></tr>
+          <tr><td><b>Commit</b></td><td><tt>#{commit}</tt></td></tr>
+          </table>
+          #{params[:footnote]}
+        COMMENT_BODY
+
+        if params[:fold]
+          "<details><summary>#{intro}</summary>\n#{body}</details>\n"
+        else
+          "<p>#{intro}</p>\n#{body}"
+        end
+      end
+
+      #####################################################
+      # @!group Documentation
+      #####################################################
+
+      def self.description
+        'Generates the nicely-formatted string providing all the details of a prototype build, ready to be used as a PR comment (e.g. via `comment_on_pr`).'
```

Comment:
Nitpick: "Generates _the_ nicely-formatted" string..." makes me think we've just been talking about it, but a reader might not have context on it when checking the documentation.

Also, is the string that is nicely formatted, or is it the HTML content it contain?

```suggestion
        'Generates a string providing all the details of a prototype build, nicely-formatted and ready to be used as a PR comment (e.g. via `comment_on_pr`).'
```